src/Progress/MVVM/ViewModelBase.cs

```diff
@@ -177,7 +41,6 @@ protected void VerifyAccess()
         [SuppressMessage("Microsoft.Design", "CA1026:DefaultParametersShouldNotBeUsed", Justification = "Set by [CallerMemberName]")]
         protected virtual void RaisePropertyChanged([CallerMemberName]string propertyName = null)
         {
-            this.VerifyAccess();
```

Comment:
This call to `VerifyAccess()` was the only actual call to any of the methods in this class.

I considered refactoring the class to pass in an `IThreadHandling` instance to do the equivalent check, but decided it wasn't worth it. The existing code has not been changed for years, and shouldn't be changed before it is dropped.

Also, we don't bother checking the thread being used in the implementation of [ViewModelBase](https://github.com/SonarSource/sonarlint-visualstudio/blob/753f8b7275d3f9d94355603ee373b79e37d40e48/src/Integration/WPF/ViewModelBase.cs#L27) in `Integration`.